webapp/lib/model.dart

```diff
@@ -128,6 +128,29 @@ List<Map> messages = [
   }
 ];
 
+List<Map> escalatesSuggestedReplies = [
+  {
+    "messages":
+      [
+        "Do you need information regarding Coronavirus?",
+        "Ma u baahantahay macluumaad ku saabsan xanuunka Koroona fayraska?",
+      ],
+    "reviewed": false,
+    "reviewed-by": "",
+    "reviewed-date": ""
+  },
+  {
+    "messages":
+      [
+        "Thanks for your message. Unfortunately we only provide information on coronavirus.a",
+        "Waad ku mahadsantahay farriintaada. Nasiib darro kaliye waxaan bixinaa macluumaad ku saabsan Koronafayraska.",
+      ],
+    "reviewed": false,
+    "reviewed-by": "",
+    "reviewed-date": ""
+  }
+];
+
 class Configuration {
```

Comment:
What do you think about we pre-initialising the attributes of this class? I think it would simplify a bit the code that defines `packageConfigurationData` below. So we would have here something like: 
```
  Map<String, TagStyle> availableTags = {};
  Map<String, TagStyle> hasAllTags = {};
  ..etc.
```

This way we ensure that none of the attributes of the class are `null`, and when we define the packages below, we can skip setting the empty maps/lists. `Open conversations` wouldn't even need any attribute setting.

Reply:
Good idea. Initialized them